package.json

```diff
@@ -162,6 +160,12 @@
                 "when": "editorTextFocus && editorLangId == lean"
             },
             {
+                "command": "lean.displayList",
+                "key": "ctrl+shift+alt+enter",
```

Comment:
I don't think these key combinations are a big issue.  The most common workflow is probably to open the info view immediately after starting vscode and then just watching the updates (at least that's what I do).  In this scenario, the keybinding is only used once per session and should not cause RSI.